datajunction-ui/src/app/services/DJService.js

```diff
@@ -12,7 +12,7 @@ export const DataJunctionAPI = {
   },
 
   logout: async function () {
-    await await fetch(`${DJ_URL}/basic/logout/`, {
+    await await fetch(`${DJ_URL}/logout/`, {
```

Comment:
2 await's needed?

Reply:
I can't remember the exact reason why I had to do this here way back. With the single await I think it was not clearing the cookies being deleted in the response from the backend.